### PR TITLE
Feature/view

### PR DESCRIPTION
**Pull request by Izebiafe**
# Pull Request: Implement Design and Styling

## Changes Made

- Implemented the design based on the sneak peek wireframes.
- Added styling to enhance the visual appearance.
- Used methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

### CODE REVIEW

File: app/views/posts/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                            </button>
+                        </h2>
+                        <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne">
+                            <div class="card-body">
+                                <!-- Single comment-->
+                                <%= render template: 'comments/new', post: @post, user: @user, class: "mb-3"%>
+                                <br/>
+                                <br/>
+                                <%if @post.recent_comment.count > 0%>
+                                    <% @post.recent_comment.each do |comment| %>
```
Comment (tugsem) [inline app/views/posts/show.html.erb:26]: - [ ] In the show page for a post, please render all of the comments as shown here: 
image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<section class="my-bio bg-secondary bg-gradient">
+  <div class="container">
+    <div class="row">
+      <%= render partial: 'user', locals: { user: @user } %>
+    </div>
+    <div class="card my-4">
+      <%= render partial: 'bio', locals: { user: @user } %>
+    </div>
+    <%= link_to "Add a new post", new_user_post_path(user_id: @user.id), class: "btn btn-primary"%>
```
Comment (tugsem) [inline app/views/users/show.html.erb]: - [ ] Please fix this error that occurs when I click a user as per the requirement.
image

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                        <h2 class="accordion-header">
+                            <button class="accordion-button" type="button" data-bs-toggle="collapse"
+                                data-bs-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+                                Comments
+                            </button>
+                        </h2>
+                        <div id="collapseOne" class="accordion-collapse collapse show" aria-labelledby="headingOne">
+                            <div class="card-body">
+                                <!-- Single comment-->
+                                <%= link_to "Add a comment", new_user_post_comment_path(post_id: post.id, user_id: @user.id), class: "btn btn-primary"%>
```
Comment (tugsem) [inline app/views/posts/index.html.erb]: - [ ] Please fix this error that occurs with the user with posts.
image

File: app/views/application/_user.html.erb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+		<div class="media-body">
+			<a href="/users/<%=user.id%>">
+				<h1 class="mt-4 text-center display-4">
+					<%= user.name%>
+				</h1>
+			</a>
+			<ul>
+				<li>
+					<h3>Number of Posts: <%= user.posts_count.present? ? user.posts_count : 0 %>
+					</h3>
```
Comment (tugsem) [inline app/views/application/_user.html.erb]: - [ ] Please move the username to the left to match the given wireframe: 
image
image

File: app/views/application/_likes.html.erb [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<%= form_for(@post.likes.new, url: user_post_likes_path(@post.author, @post)) do |form| %>
```
Comment (tugsem) [inline app/views/application/_likes.html.erb:1]: - [ ] I can't display the views as this method throws an error. Kindly fix this.
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @Izebiafe  👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️linters are passing
✔️correct use of gitflow
✔️user index page is created successfully
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Strangeal — APPROVED** [review]
Hi @Izebiafe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉 



## Highlights
- Adjusted `design` to match wireframe ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**